### PR TITLE
Make `copy-file` feature work on gists

### PR DESCRIPTION
**Pull request by kidonng**
I don't know if this works before, but from `features.isGist` I guess it is expected to, so here it is.

# Test

https://gist.github.com/KomaliVuddanti/f062a78b4fff3ee001529efe2a468375

### CODE REVIEW

File: source/features/copy-file.tsx [hunk 1]
```diff
@@ -5,16 +5,22 @@ import copyToClipboard from 'copy-text-to-clipboard';
 import features from '../libs/features';
 
 function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
-	const file = button.closest('.Box');
+	const selector = features.isGist() ? '.js-gist-file-update-container' : '.Box';
+	const file = button.closest(selector);
```
Comment (fregante) [inline source/features/copy-file.tsx]: No need to is isGist, just merge the selectors

File: source/features/copy-file.tsx [hunk 2]
```diff
@@ -5,16 +5,22 @@ import copyToClipboard from 'copy-text-to-clipboard';
...
 	copyToClipboard(content);
 }
 
 function renderButton(): void {
-	for (const blameButton of select.all('[data-hotkey="b"]')) {
-		blameButton
+	const selector = features.isGist() ? '.file-actions .btn' : '[data-hotkey="b"]';
+	for (const button of select.all(selector)) {
+		if (features.isGist()) {
+			button.classList.add('BtnGroup-item');
```
Comment (fregante) [inline source/features/copy-file.tsx]: You can use the groupButtons function to do this automatically, without conditions. Call it after the prepend

Comment (kidonng) [reply source/features/copy-file.tsx]: Have I done it right?

Comment (fregante) [reply source/features/copy-file.tsx]: You can leave the old code (without the extra const) and use `groupSiblings(button)` instead, I think.

Comment (kidonng) [reply source/features/copy-file.tsx]: I thought you were literally talking `groupButtons` function.

File: source/features/copy-file.tsx [hunk 3]
```diff
@@ -3,18 +3,20 @@ import select from 'select-dom';
...
 	const content = select.all('.blob-code-inner', file!)
 		.map(({innerText: line}) => line === '\n' ? '' : line) // Must be `.innerText`
 		.join('\n');
 	copyToClipboard(content);
 }
 
 function renderButton(): void {
-	for (const blameButton of select.all('[data-hotkey="b"]')) {
-		blameButton
+	const selector = features.isGist() ? '.file-actions .btn' : '[data-hotkey="b"]';
```
Comment (jerone) [inline source/features/copy-file.tsx]: Can these selectors also be merged?

File: source/features/copy-file.tsx (hunk 2 above)
Comment (fregante) [reply source/features/copy-file.tsx]: Sorry about the confusion 😁

File: source/features/copy-file.tsx (hunk 3 above)
Comment (fregante) [reply source/features/copy-file.tsx]: This isn’t resolved yet. By “merging” he means what you did with the previous selector where I suggested to drop isGist

Comment (kidonng) [reply source/features/copy-file.tsx]: Sorry I always forget its parameter can be multiple selectors

**Pull request merged**